enterprise/admin/views.py

```diff
@@ -62,20 +64,56 @@ def _build_context(self, request, customer_uuid):
         """
         # TODO: pylint acts stupid - find a way around it without suppressing
```

Comment:
It looks like this has been a [known issue with pylint-django](https://github.com/landscapeio/pylint-django/pull/70) for [quite a while.](https://github.com/landscapeio/pylint-django/issues/53) Not much we can do at this point from our end other than continue to tag the specific cases that need them. I don't think this is tech debt on our end, really, so I'd be up for removing that TODO.